### PR TITLE
Bugs/zero x prefixing 178965517

### PR DESCRIPTION
**Pull request by rosalinekarr**
Problem
=======
Ethers requires a 0x prefix on all hexadecimal values, and our code is inconsistent in hexadecimal prefixes.
[#178965517](https://www.pivotaltracker.com/story/show/178965517)

Solution
========
Update all hexadecimal values everywhere to start with a "0x"

### CODE REVIEW

**Review by enddynayn — APPROVED** [review]
fabulous 👍

File: src/core/utilities/hash.test.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import { hash } from "./hash";
+
+describe("hash", () => {
+  describe("hash", () => {
```
Comment (acruikshank) [inline src/core/utilities/hash.test.ts:4]: Don't think you need this second describe.

**Review by acruikshank — APPROVED** [review]
Looks good.

File: src/core/utilities/hash.test.ts (hunk 1 above)
Comment (rosalinekarr) [reply src/core/utilities/hash.test.ts]: Fixed

**Pull request merged**